### ISSUE TITLE
Get rid of lazyload class

### ISSUE THREAD

**Issue by Zmove** [aFarkas/lazysizes#366]
Hello,

Love this plugin, however I'm not fan of adding a class on each imagess to apply laryload on them.

It could be done by targeting image with data-src or data-srcset directly.

**Comment by jurajkapsz** [comment]
My two cents on this:

1. `class-based` selectors [outperforms](http://stackoverflow.com/a/12497267) `data attribute` matching,
1.  lazyload class name can be [reconfigured](https://github.com/aFarkas/lazysizes/blob/gh-pages/README.md) via JS API options to e.g. "lazy" or "lz" and so on if the size of html code matters, or if it's easier to be used everytime,
1. I like to style my lazy loaded images, mostly to add opacity transition effects.

So I find it beneficial the way it is now.

**Comment by jhabdas** [comment]
> It could be done by targeting image with data-src or data-srcset attributes directly.

Yeah, but what about scripts, iframes? That kind of stuff adds code complexity and starts to make assumptions about data structure that could have unexpected consequences. For example, WordPress may add `data-src` AND `src` automatically. How would you surmise to handle a situation like that if you didn't want to lazyload an image?
